### PR TITLE
Added some LEDs

### PR DESCRIPTION
**Pull request by fkbreitl**
I have added some LEDs - most important BEAT SYNC!
It didn't get the hotcues to work.

### CODE REVIEW

File: DDJ-200.midi.xml [hunk 1]
```diff
@@ -837,6 +837,89 @@
...
             </control>
         </controls>
-        <outputs/>
-    </controller>
+
+        <outputs>
+            <output>
+                <group>[Channel1]</group>
+		<key>play</key>
+                <description>Deck 1: Play button</description>
```
Comment (dan-giddins) [inline DDJ-200.midi.xml]: Looks good, but there is already a mapping for these buttons in this file. It's probably better if you add the extra on/off/etc... tags to the existing mappings

Comment (fkbreitl) [reply DDJ-200.midi.xml]: Maybe now I understand what you are suggesting.
But to my understanding of the XML mapping it is not possible to add on/off sections to the existing section.
LEDs need a separate output section.

**Comment by fkbreitl** [comment]
I got the hotcue LEDs working using `hotcue_1_enabled` as key thanks to the help of [Mixxx developer hlzhs](https://mixxx.discourse.group/t/how-to-switch-on-hotcue-leds/19997/2?u=aldous).

File: DDJ-200.midi.xml (hunk 1 above)
Comment (dan-giddins) [reply DDJ-200.midi.xml]: Yes I see what you mean now, this is actually a different mapping isn't it, my bad!

**Comment by dan-giddins** [comment]
This all looks good! I would like to test it out myself before I merge into master, but I will do this the next time I am at home, and will do this before I do the other work I need to do to merge my main MIXXX PR, such that your changes will be part of that PR.

**Review by dan-giddins — APPROVED** [review]

**Comment by dan-giddins** [comment]
I've decided just to merge it, as a PR should be about the quality of the code, not testing it :)
I will get this code into my main mixxx PR as well when I have the chance.

I will tag you in it, as there is some work to do on the wiki which you might be able to help me with!

**Comment by fkbreitl** [comment]
Thank you very much for pulling.
In the meantime I have also tried to get the master button (headMix) working:
https://mixxx.discourse.group/t/how-to-map-the-mastercue-button/20001

However, this is more difficult especially after discovering that the controller gets unresponsive after pressing some button:
https://bugs.launchpad.net/mixxx/+bug/1898375
https://bugs.launchpad.net/mixxx/+bug/1898374

So in principle I have created a function, but I can't test it, since the controller behaves strange sometimes.
So when these other problems are fixed, getting the button working should be easier.

**Pull request merged**